mesop/component_helpers/style.py

```diff
@@ -22,6 +22,46 @@
   "center",
   "start",
   "end",
+  "flex-start",
+  "flex-end",
+  "self-start",
+  "self-end",
+  "baseline",
+  "first baseline",
+  "last baseline",
+  "safe center",
+  "unsafe center",
+  "inherit",
+  "initial",
+  "revert",
+  "revert-layer",
+  "unset",
+]
+ItemJustifyValues = Literal[
+  "normal",
+  "stretch",
+  "center",
+  "start",
+  "end",
+  "flex-start",
+  "flex-end",
+  "self-start",
+  "self-end",
+  "left",
+  "right",
+  "baseline",
+  "first baseline",
+  "last baseline",
+  "safe center",
+  "unsafe center",
+  "legacy left",
```

Comment:
I would remove the legacy values - I can't really think of why someone would use it in Mesop (IIUC, the web has this for legacy compatibility reasons)

Reply:
Good call. I was waffling on keeping/leaving those. Ended up keeping them, I think you make good points that Mesop users likely shouldn't need to use legacy properties unless they were copying some bad code.